ape_infura/__init__.py

```diff
@@ -5,7 +5,6 @@
 NETWORKS = {
     "ethereum": [
         "mainnet",
-        "goerli",
```

Comment:
Don't think this is deprecated yet, but not a huge deal to have to pin below the release containing this